templates/CRM/Admin/Page/Job.tpl

```diff
@@ -39,14 +39,14 @@
             <th ></th>
         </tr>
         {foreach from=$rows item=row}
-        <tr id="job-{$row.id}" class="crm-entity {cycle values="odd-row,even-row"} {$row.class}{if NOT $row.is_active} disabled{/if}">
+        <tr id="job-{$row.id}" class="crm-entity {cycle values="odd-row,even-row"} {if !empty($row.class)}{$row.class}{/if}{if NOT $row.is_active} disabled{/if}">
```

Comment:
LOL, I remember back in the days of CSSv1 when there were no selectors for `:nth-child(even)` and `:nth-child(odd)` - apparently our .tpl files do too!